### PR TITLE
tree-wide: fix "the the" and "that that"

### PR DESCRIPTION
**Pull request by yuwata**
Some of the changes in the second commit may not be necessary. Not sure...

### CODE REVIEW

File: docs/CONTAINER_INTERFACE.md [hunk 1]
```diff
@@ -364,7 +364,7 @@ If you write software that wants to detect whether it is run in a container,
 please check `/proc/1/environ` and look for the `container=` environment
 variable. Do not assume the environment variable is inherited down the process
 tree. It generally is not. Hence check the environment block of PID 1, not your
-own. Note though that that file is only accessible to root. systemd hence early
+own. Note though that the file is only accessible to root. systemd hence early
```
Comment (keszybz) [inline docs/CONTAINER_INTERFACE.md]: Either way is OK, but maybe change to "this file"?

File: src/basic/copy.c [hunk 2]
```diff
@@ -1103,7 +1103,7 @@ int copy_file_atomic_full(
         assert(from);
         assert(to);
 
-        /* We try to use O_TMPFILE here to create the file if we can. Note that that only works if COPY_REPLACE is not
+        /* We try to use O_TMPFILE here to create the file if we can. Note that it only works if COPY_REPLACE is not
```
Comment (keszybz) [inline src/basic/copy.c]: "It" refers to "file", so this sentence doesn't make sense now. "Note that this only work ..." maybe?

File: src/shared/loop-util.c [hunk 3]
```diff
@@ -232,7 +232,7 @@ static int loop_configure(
          * ioctl can return EAGAIN in case we change the lo_offset field, if someone else is accessing the
          * block device while we try to reconfigure it. This is a pretty common case, since udev might
          * instantly start probing the device as soon as we attach an fd to it. Hence handle it in two ways:
-         * first, let's take the BSD lock that that ensures that udev will not step in between the point in
+         * first, let's take the BSD lock so that it ensures that udev will not step in between the point in
```
Comment (keszybz) [inline src/shared/loop-util.c]: "take the BSD lock to ensure"

**Review by keszybz — CHANGES_REQUESTED** [review]

**Comment by yuwata** [comment]
@keszybz Thank you for you review. Updated. PTAL.

**Pull request merged**